tests/test_spdx_matcher.py

```diff
@@ -40,10 +41,12 @@
 issues with regexp performance such as back tracking
 """
 
-with open("tests/APACHE.txt", mode="rt", encoding="utf-8") as af:
+current_dir = Path(__file__).parent
+
+with open(f"{current_dir}/APACHE.txt", mode="rt", encoding="utf-8") as af:
     APACHE2 = af.read()
 
-with open("tests/CHALLENGING.txt", mode="rt", encoding="utf-8") as af:
+with open(f"{current_dir}/CHALLENGING.txt", mode="rt", encoding="utf-8") as af:
```

Comment:
To make test file can be run in ide or in tests directory by `python test_spdx_matcher.py`